.github/actions/spelling/expect.txt

```diff
@@ -1,5 +1,6 @@
 abcd
```

Comment:
cspell dictionaries were changed so I think this resulted in more words needing to be added to the `expect.txt`. Updated by running the `apply.pl` suggestion from the `check-spelling` results.